source/overview/product.rst

```diff
@@ -2,70 +2,154 @@
 Product Overview
 ============================
 
-**Mattermost** empowers organizations to achieve their highest priorities through modern, enterprise-grade communication. Enjoy all the productivity benefits of workplace messaging across web, mobile and PC, with unlimited archiving, search and integrations in a single-tenant, private cloud solution under IT control.
+Mattermost allows organizations to achieve their highest goals through enterprise-grade communication, 
```

Comment:
We should talk to marketing team to see if they have a newer sentence they want to use here

Reply:
Sure. Sounds good. I will share my thinking. Every word has two parts, the intended and the opposite, so when you use a word like empower it suggests the people are at present unempowered or disempowered. It's also an extremely political word. It makes me think of abused banana pickers here in Costa Rica.